neuro_data/static_images/requests.py

```diff
@@ -0,0 +1,26 @@
+from os import sched_getaffinity
```

Comment:
What is this requests.py module for?

Reply:
This is for providing the `content` attribute for the `DataConfig` part tables without putting all data in the source code.